class/practice_6.1/message_generator.cpp

```diff
@@ -0,0 +1,42 @@
+#include "message_dispatcher.hpp" 
+
+void generateMessages(MessageDispatcher& dispatcher, const std::string& threadName) {
+    std::random_device rd;
+    std::mt19937 gen(rd());
+    std::uniform_int_distribution<> counterDist(1, 10);
+    std::uniform_real_distribution<> doubleDist(0.0, 10.0);
+
+    while (true) {
+        int messageType = gen() % 3;
+
+        switch (messageType) {
+            case 0: {
+                std::string text = "Green message: ";
```

Comment:
maybe we can split this function to few part for example greenHandler(), blueHandler and so on